src/index.html

```diff
@@ -1,14 +1,50 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>Enclosures</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Enclosures</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Enclosures</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <div class="bigBlock"
+         id="qa-block"
+    >
+      <div class="middleBlock">
+        <div
+        class="block-with-info"
+        >
```

Comment:
no need to push attributes to the newline if you have less then 3 of them, fix everywhere
```suggestion
    <div class="bigBlock" id="qa-block">
      <div class="middleBlock">
        <div class="block-with-info">
```